pingcap/tiup
# cluster: add role and node filter for `check` subcommand

### What problem does this PR solve? 
Close #1022 

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Related changes

 - Need to update the documentation


Release notes:

```release-note
cluster: support filtering node and role in `check` subcommand
```
